fvtr/fvtr.sh

```diff
@@ -37,6 +38,7 @@ function usage () {
 	echo "Usage: fvtr [-fl] <configfile> [test1 test2 ...]"
 	echo "       -f Force the test(s) to be run"
 	echo "       -l Output is sent to the console"
+	echo "       -d Print detailed log for errors"
```

Comment:
-l sends absolutely all logs to console. -d only sends logs of tests that actually failed. I did notice that I made a mistake, though: -l and -d should be mutually exclusive. I'll fix that in the next commit.